ckan/views/group.py

```diff
@@ -827,6 +827,47 @@ def post(self, id, group_type, is_organization, data=None):
         return h.redirect_to(u'{}.bulk_process'.format(group_type), id=id)
 
 
+def changes(id, group_type, is_organization):
+    '''
+    Shows the changes to an organization in one particular activity stream item.
+    '''
+    activity_id = id
+    context = {
+        u'model': model, u'session': model.Session,
+       u'user': g.user, u'auth_user_obj': g.userobj
+    }
+    try:
+        activity_diff = get_action(u'activity_diff')(
+            context, {u'id': activity_id, u'object_type': u'group',
+                      u'diff_type': u'html'}
```

Comment:
There is a syntax error which causes ckan not to start in tests.
```suggestion
                      u'diff_type': u'html'})
```

Reply:
@Zharktas , I have corrected the syntax error in commit https://github.com/ckan/ckan/pull/5912/commits/50fa7dc62ee48a882791c9f0f32213b218b78652